src/app/components/SummaryHighlight/SummaryHighlight.tsx

```diff
@@ -0,0 +1,40 @@
+import classes from './SummaryHighlight.module.css';
+
+type Options = {
+  hour: 'numeric';
+  minute: 'numeric';
+  second: 'numeric';
+  hour12: boolean;
+  timeZone: string;
+};
```

Comment:
`'numeric'` is not `number`. It's a string of the word `numeric`.
Thus, these `options` are always the string `"numeric"`.